src/app/components/modal/modal-filter.component.ts

```diff
@@ -109,68 +114,78 @@ export class ModalFilterComponent extends DynamicComponent {
     }
 
     positionStartModal() {
-        return {
-            top:
-                this.getElementPosition().top +
-                this.getElementPosition().height +
-                "px",
-        };
+        const elementPosition = this.getElementPosition();
+        if (elementPosition?.top && elementPosition?.height) {
+            return {
+                top: elementPosition.top + elementPosition.height + "px",
+            };
+        }
+        return {};
     }
 
     styleFilterModal() {
-        const styleModal: any = {};
-
+        const styleModal: ObjectInterface = {};
```

Comment:
Try using `CSSStyleDeclaration` for styles

Reply:
Fixed